packages/uswds-core/src/styles/mixins/typography/typeset.scss

```diff
@@ -39,10 +39,6 @@ Sets:
   @include u-margin-y(0);
   clear: both;
 
-  * + & {
-    margin-top: 1.5em; // TODO: add as var to settings?
-  }
-
```

Comment:
`* + &` gets confused when added to mixins, so moved these items directly inside the style rules.